### PR TITLE
Allow `install_sct.bat` file execution from double click

### PR DESCRIPTION
**Pull request by RignonNoel**
#### GitHub

- [x] I've given this PR a concise, self-descriptive, and meaningful title
- [x] I've linked relevant issues in the PR body
- [x] I've applied [the relevant labels](https://intranet.neuro.polymtl.ca/geek-tips/contributing#pr-labels-a-href-pr-labels-id-pr-labels-a) to this PR
- [x] I've applied a [release milestone](https://github.com/spinalcordtoolbox/spinalcordtoolbox/milestones) (major, minor, patch) in line with [Semantic Versioning guidelines](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Misc%3A-Creating-a-new-release#convention-for-naming-releases) 
- [x] I've assigned a reviewer

#### PR contents

- [x] I've consulted [SCT's internal developer documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki) to ensure my contribution is in line with any relevant design decisions
- [x] I've added [relevant tests](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Tests) for my contribution
- [x] I've updated the [relevant documentation](https://github.com/spinalcordtoolbox/spinalcordtoolbox/wiki/Programming%3A-Documentation) for my changes, including argparse descriptions, docstrings, and ReadTheDocs tutorial pages

## Description
Allow `install_sct.bat` to be usable from a simple double click in place of a command line execution. It will make it a lot more easy for windows user that don't have a lot of technical knowledge.

## Linked issues
#3762

### CODE REVIEW

File: install_sct.bat [hunk 1]
```diff
@@ -1,92 +1,93 @@
...
-
-rem Go to user's home directory
-pushd %HOMEPATH%
-
-rem Check to see if we're going to git clone into an existing installation of SCT
-if exist spinalcordtoolbox\ (
-  echo ### Previous spinalcordtoolbox installation found at %HOMEPATH%\spinalcordtoolbox.
-  rem NB: The rmdir command will output 'spinalcordtoolbox\, Are you sure (Y/N)?', so we don't need our own Y/N prompt
-  rem     We also use "echo set /p=" here in order to make sure that Y/N text is output on the same line.
-  echo|set /p="### Continuing will overwrite the existing installation directory " || goto error
```
Comment (RignonNoel) [inline install_sct.bat:22]: I don't know what was the goal of the `|| goto error` on a simple echo but for me the problem was there, without it the bat file work like a charm

Comment (RignonNoel) [reply install_sct.bat:22]: If you have some knowledge about that @joshuacwnewton

File: install_sct.bat [hunk 2]
```diff
@@ -1,92 +1,93 @@
...
+:error
+set cached_errorlevel=%errorlevel%
+echo Failed with error #%cached_errorlevel%.
+
+:exit
+if "%cached_errorlevel%"=="" set cached_errorlevel=0
+popd
+where deactivate
+if %errorlevel% EQU 0 call deactivate
+PAUSE
```
Comment (RignonNoel) [inline install_sct.bat:92]: Decided to use a `PAUSE` that add a beautiful end of script:

```
Press any key to continue . . .
```

If you prefer to have nothing i can change it.

File: install_sct.bat (hunk 1 above)
Comment (joshuacwnewton) [reply install_sct.bat:22]: I don't think there was any big reasoning here -- I agree that it's fine to omit the `|| goto error` for the `echo`. :)

**Review by joshuacwnewton — APPROVED** [review]
Apart from the line-endings, this LGTM! 

(Back to CRLF for `.bat` files, then?)

**Comment by RignonNoel** [comment]
> (Back to CRLF for `.bat` files, then?)

Yes, as we discussed it will be better to make it work on Windows.

I will check how to disable the auto-format on my device and push with CRLF

**Pull request merged**